index.d.ts

```diff
@@ -895,6 +896,34 @@ declare module "axoncore" {
         public delete(mID: string): Collection<Message>;
     }
 
+    interface rCollectorOptions {
```

Comment:
Dev branch for this requires interface names to be PascalCased. I know it's a bit naff that you're doing this on master but leaving this here fyi
```suggestion
    interface CollectorOptions {
```